plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -340,7 +347,27 @@ public Instance startService(String namespace,
         }
     }
 
-    @Override
+    private void prepareContainer(String containerId, LineConsumer machineLogger) throws MachineException {
```

Comment:
name `prepareContainer` doesn't say much about this method. What about `changeWorkingDirOwner`?